collectors/proc.plugin/proc_pagetypeinfo.c

```diff
@@ -222,7 +222,7 @@ int do_proc_pagetypeinfo(int update_every, usec_t dt) {
 
 
         // Per-Numa Node & Zone & Type (full detail). Only if sum(line) > 0
-        st_nodezonetype = callocz(pagelines_cnt, sizeof(RRDSET));
+        st_nodezonetype = callocz(pagelines_cnt, sizeof(RRDSET *));
```

Comment:
I prefer:
`st_nodezonetype = callocz(pagelines_cnt, sizeof(*st_nodezonetype));`
cause this way you don't have to go lookup the definition. Also, if the definition changes you don't have to remember changing the allocation as well.

Reply:
I understand your point, but case the variable type is changed, a lot of other things will be changed in this code.